hawc/tools/docx_utils/reports.py

```diff
@@ -0,0 +1,73 @@
+import abc
+import os
+from io import BytesIO
+
+from docx import Document
+from docx.enum.section import WD_ORIENT
+from docx.shared import Inches
+
+
+class DOCXReport(object):
```

Comment:
remove `(object)` parent class for this and other-things in docx_utils, old python2 convention.

Looking forward to adding type annotations to this once we start using it.